deploy/build-container.sh

```diff
@@ -117,20 +112,20 @@ ctr=$(sudo buildah from \
 
 [[ -z "${ctr}" ]] && err "error obtaining base container" && exit 1
 
-buildah run ${ctr} zypper install -y python38-pip || exit 1
-buildah run ${ctr} pip install fastapi uvicorn aiofiles || exit 1
+sudo buildah run ${ctr} zypper install -y python38-pip || exit 1
```

Comment:
this removes the possibility of running this script in an unprivileged podman environment. I'd keep the `--no-sudo` option and add a `do_sudo()` function or something that would call the command with `sudo` or not depending on the option.